The conditional load needs to be made part of an IT block on Thumb2
cores.

Signed-off-by: Will Newton <will.newton at linaro.org>
---
 libc/sysdeps/linux/arm/clone.S | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libc/sysdeps/linux/arm/clone.S b/libc/sysdeps/linux/arm/clone.S
index 98dde22..e73ddb6 100644
--- a/libc/sysdeps/linux/arm/clone.S
+++ b/libc/sysdeps/linux/arm/clone.S
@@ -110,6 +110,7 @@ __clone:
 	ldr	r4, [sp, #12]
 	DO_CALL (clone)
 	movs	a1, a1
+	IT(t, ne)
 	ldmnefd	sp!, {r4}
 	blt	__error
 	IT(t, ne)
-- 
1.8.1.4
